pkg/status/console_test.go

```diff
@@ -194,54 +201,3 @@ func Test_consoleGetKind(t *testing.T) {
 		})
 	}
 }
-
-func Test_statusUnknown(t *testing.T) {
```

Comment:
Why is this test removed?

Reply:
We don't want a statusUnknown error, it would error out to statusUnkown usually before adding the Console disabled status. I believe we want to have it have the console disabled status in replace of statusUnknown. I noticed that the error I was sending to statusUnkown method on `apierrors.IsNotFound(err)` would be true always when console was not found, and that is why we are adding this custom status instead. So I deleted the check that sent to unknownStatus, therefore we don't need to test it anymore

Reply:
Gotcha. I thought there was a statusUnknown function missing a test.